recipes/proposal/all/conandata.yml

```diff
@@ -35,3 +35,6 @@ sources:
   "7.4.2":
     url: "https://github.com/tudo-astroparticlephysics/PROPOSAL/archive/refs/tags/7.4.2.tar.gz"
     sha256: "f0db44c96a80a6ce3dda02c598574f5f0209376bd2c6c176797710da8eb3e108"
+  "7.5.0":
```

Comment:
Please add new versions on the top, the backend like to build in order and it will give faster results in the most general case :)

Reply:
So for the next version update, should I completely reverse the order, or just add the next version on top of the existing file?

Reply:
Ideally reverse it, but I would be happy if it just at the top :)